pages/blog/typescript-types.mdx

```diff
@@ -0,0 +1,145 @@
+import withArticle from '../../ui/blog/article'
+export const meta = {
+  title: 'Back to front: TypeScript union types and how to use GraphQL Codegen to create type-safe',
+  author: 'tuvalSimha',
+  tags: ['graphql', 'typescript', 'codegen'],
+  date: '2022-07-20',
+  description: 'TypeScript union types in use GraphQL schema, and how to use GraphQL Codegen to create type-safe.',
+  image: '/blog-assets/graphql-code-generator/cover.png'
```

Comment:
Let's use a cover image with the article's title in it to make it more visible on the website and when shared on social media